clustering/ee/cache/src/main/java/org/wildfly/clustering/ee/cache/scheduler/LocalScheduler.java

```diff
@@ -115,15 +115,13 @@ public void accept(Map.Entry<T, Instant> entry) {
     }
 
     @Override
-    public synchronized void close() {
-        WildFlySecurityManager.doPrivilegedWithParameter(this.executor, DefaultExecutorService.SHUTDOWN_ACTION);
-        if (this.future != null) {
+    public void close() {
+        WildFlySecurityManager.doPrivilegedWithParameter(this.executor, DefaultExecutorService.SHUTDOWN_NOW_ACTION);
```

Comment:
@pferraro So it's ok if any executing run() method gets interrupted and any scheduled but not yet executed run() doesn't get called?

I figure yes, but I'm asking because this seems to be the key change in the PR.

Reply:
Correct.